### DISCUSSION TITLE
Configure HTTPS

### DISCUSSION THREAD

**Discussion by carbotechTe** [globaleaks/globaleaks-whistleblowing-software#3454] [General]
Good morning,
i tried setting up https in my server.

But during the procedure i have this error:
`Version: 4.11.5

AttributeError Attribute not found.

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/twisted/internet/defer.py", line 1418, in _inlineCallbacks
    result = g.send(result)

  File "/usr/lib/python3/dist-packages/globaleaks/rest/api.py", line 449, in concludeHandlerFailure
    self.handle_exception(err, request)

  File "/usr/lib/python3/dist-packages/globaleaks/rest/api.py", line 276, in handle_exception
    e.tid = request.tid

  File "/usr/lib/python3/dist-packages/josepy/util.py", line 179, in __setattr__
    raise AttributeError("can't set attribute")

AttributeError: can't set attribute`

Where did i go wrong?

Thank you

**Comment by carbotechTe** [comment]
When i go to auto-generate https i just see the spinner spinning endlessly. If i reload the page i get the error posted before.

**Comment by evilaliv3** [comment]
Hello @carbotechTe; this seems to be a software failure not depending from you.

May i ask you:
- on which os/version is GlobaLeaks installed?
- did you configure a DNS entry pointing your domain to the public IP of the server?
- is the server publicly reachable on both ports 80 and 443 ?

**Comment by carbotechTe** [comment]
Hello,
- Ubuntu 20.04.6 LTS OS
- DNS Ok
- Port 80 and 443 Ok

Responds out of 80 if i try https(443) the browser blocks "FireFox SSL_ERROR_NO_CYPHER_OVERLAP - Chrome ERR_SSL_VERSION_OR_CIPHER_MISMATCH"

**Comment by carbotechTe** [comment]
Hello @evilaliv3,
i tried auto-generating https.
The app didn't crash and i didn't get any error mails.
But if i try to access in https it doesn't work.

Does this app auto redirect to https?

Thank you
